site/content/docs/user/resources.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Resources"
+menu:
+  main:
+    parent: "user"
+    identifier: "resources"
+    weight: 4
+---
+# Additional Resources
+
+Here are some additional resources for learning about kind and how to use it.
+
+## KubeCon Talks
+
+The authors have given the following talks relating to kind:
+
+### Testing your K8s apps with KIND - Benjamin Elder & James Munnelly
+
+At KubeCon EU 2019 we spoke about [kind and testing your Kubernetes Applications][testing-k8s-apps-with-kind].
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/8KtmevMFfxA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
TODO: add a class to these and look at styling them to auto-size better instead of the fixed size. this is just the standard embed code currently.

NOTE: these are the GDPR compliant (in theory) / no-cookie / enhanced privacy mode embeds.